### DISCUSSION TITLE
Hi and Help. I have a ITEAD itdb02 MEGA sheild and I want to use it with GUISLICE,

### DISCUSSION THREAD

**Discussion by marcwolf1960** [ImpulseAdventure/GUIslice#379] [Q&A]
Hi.
I can get the unit working with UTFT with these parameters **UTFT myGLCD(ITDB32S,38,39,40,41);**
 I can get the touch screen working and the SD card with standard libraries.
However when ever I try and get this card running with GUISLICE I just get a blank screen.
The chipset the TFT uses is a SSD1289

Many many thanks
Dave

**Comment by ImpulseAdventure** [comment]
Hi Dave --

What config file are selecting in `GUIslice_config.h`?
- Can you confirm that you have tried `ard-shld-utft-notouch.h`?

In SECTION 2, you should try the following settings to match your UTFT initialiation:
```c++
#define DRV_DISP_UTFT_INIT  ITDB32S,38,39,40,4
```

Also, please adjust your config to set DEBUG_ERR to 2 and copy here any messages you see on the serial monitor at startup.

Thanks
